apps/astarte_appengine_api/lib/astarte_appengine_api/config.ex

```diff
@@ -279,6 +279,26 @@ defmodule Astarte.AppEngine.API.Config do
             ]
           ]
         ]
+
+      "docker-compose" ->
+        [
+          data_updater_plant: [
+            strategy: Elixir.Cluster.Strategy.DNSPoll,
+            config: [
+              polling_interval: 5_000,
+              query: "astarte-data-updater-plant",
+              node_basename: "astarte_data_updater_plant"
+            ]
+          ],
+          appengine_api: [
+            strategy: Elixir.Cluster.Strategy.DNSPoll,
+            config: [
+              polling_interval: 5_000,
+              query: "astarte-appengine-api",
+              node_basename: "astarte_appengine_api"
+            ]
```

Comment:
I think appengine api instances only need to find data updater plants to make RPC requests, no need to find other appengine instances